start/src/main/java/org/apache/accumulo/start/classloader/vfs/ContextManager.java

```diff
@@ -189,13 +189,13 @@ public ClassLoader getClassLoader(String contextName) throws FileSystemException
     }
   }
 
-  public void removeUnusedContexts(Set<String> inUse) {
+  public void removeUnusedContexts(Set<String> configuredContexts) {
```

Comment:
badly named.....we are not removing unused contexts here, we are removing from the list of unused contexts.  Perhaps simply calling this removeConfiguredContexts or removeUsedContexts would make more sense.